opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -123,28 +123,39 @@ message MetricDescriptor {
     // INVALID_TYPE is the default Type, it MUST not be used.
     INVALID_TYPE = 0;
 
-    // INT64 values are represents as signed 64-bit integers.
+    // INT64 values are signed 64-bit integers.
     //
     // A Metric of this Type MUST store its values as Int64DataPoint.
     INT64 = 1;
 
-    // DOUBLE values are represents as double-precision floating-point
-    // numbers.
+    // MONOTONIC_INT64 values are monotonically increasing signed 64-bit
+    // integers.
+    //
+    // A Metric of this Type MUST store its values as Int64DataPoint.
+    MONOTONIC_INT64 = 2;
```

Comment:
Should we document the interaction with temporality?

Reply:
I'm not following, is that not already documented in and on the `Temporality` message itself already?

Reply:
I agree with this change about Monotonic representation. I've come to disagree with use of "Temporality" and do not believe we should use any of the terms Instantaneous, Cumulative, or Delta directly in the protocol.

I have outlined my latest opinion here: https://github.com/open-telemetry/opentelemetry-proto/issues/125#issuecomment-635196903

There I used the terms "Collective vs Additive" to distinguish two kinds of aggregation, but later debate has nixed the word "Collective". See https://github.com/open-telemetry/opentelemetry-specification/issues/625#issuecomment-636119812

I think we should replace the entire section on temporality with a two-state enum: Grouping or Additive.

Separately, also stated in the comment on proto issue 125, I think we should use the start and end time to convey whether reported data is delta or cumulative.

Reply:
> I agree with this change about Monotonic representation. I've come to disagree with use of "Temporality" and do not believe we should use any of the terms Instantaneous, Cumulative, or Delta directly in the protocol.
> 
> I have outlined my latest opinion here: [#125 (comment)](https://github.com/open-telemetry/opentelemetry-proto/issues/125#issuecomment-635196903)
> 
> There I used the terms "Collective vs Additive" to distinguish two kinds of aggregation, but later debate has nixed the word "Collective". See [open-telemetry/opentelemetry-specification#625 (comment)](https://github.com/open-telemetry/opentelemetry-specification/issues/625#issuecomment-636119812)
> 
> I think we should replace the entire section on temporality with a two-state enum: Grouping or Additive.
> 
> Separately, also stated in the comment on proto issue 125, I think we should use the start and end time to convey whether reported data is delta or cumulative.

The more I've thought about the Temporality the more I think I agree with this. It seems like even with directly specifying it you should still be checking (start, end) against last seen to determine if the interval was reset (cumulative) or data was lost (delta).

I'm interested to talk more with @jkwatson about this though.

Reply:
Is MONOTONIC_SUM_INT64 better name?

Reply:
Probably not... Let's keep as it is.